### ISSUE TITLE
Некорректная сборка системы

### ISSUE THREAD

**Issue by Linuxoid85** [Linux4Yourself/book#1038]
Во время сборки системы со следующей конфигурацией:

- SysVInit;
- CDS;
- LX4U 2.0;

Ошибок во время сборки пакетов нет. Однако, система отказывается загружаться дальше ядра. После него, как правило, чёрный экран, а система не реагирует ни на какие комбинации клавиш. Ошибок в логах или чего-то странного нет, поэтому требуется проверить сборочные инструкции всех обновлённых в рамках релиза пакетов на наличие ошибок.

**Comment by Linuxoid85** [comment]
UPD. С ядром, вероятно, проблем нет, ибо при использовании нескольких разных конфигов и версий ядра проблема остаётся.

Вероятно, следует проверить инструкции пакетов, обновлённых в начале декабря, с того времени иногда возникали некоторые проблемы.

**Comment by makpif** [comment]
> Ошибок в логах или чего-то странного нет

Их все равно стоит выложить хотя бы что бы было понятно на какой стадии ошибка

**Comment by Linuxoid85** [comment]
> > Ошибок в логах или чего-то странного нет
> 
> Их все равно стоит выложить хотя бы что бы было понятно на какой стадии ошибка

Да. Через некоторое время вместе с логами загрузки выложу результаты тестов каждого пакета.
